deployment/src/main/java/io/quarkiverse/quinoa/deployment/framework/override/NextFramework.java

```diff
@@ -14,6 +14,7 @@
 public class NextFramework extends GenericFramework {
 
     private static final Logger LOG = Logger.getLogger(NextFramework.class);
+    private static final String OUTPUT = "export";
```

Comment:
where is this used?

Reply:
This is just a small thing. I can remove it if it's not consistent with the code style. I just personally think that in the future, if the code for this gets bigger and there needs to be a change to this, you can change it at the top of the file and won't need to search for it.

Reply:
all good could you call it `EXPECTED_OUTPUT_VALUE`